Framework/Algorithms/src/GetDetectorOffsets.cpp

```diff
@@ -111,10 +111,10 @@ void GetDetectorOffsets::exec() {
   Progress prog(this, 0, 1.0, nspec);
   auto &spectrumInfo = maskWS->mutableSpectrumInfo();
   PARALLEL_FOR_IF(Kernel::threadSafe(*inputW))
-  for (int wi = 0; wi < nspec; ++wi) {
+  for (size_t wi = 0; wi < nspec; ++wi) {
```

Comment:
OpenMP 2.0 (and therefore MSVC) doesn't allow unsigned OpenMP index variables. Can you use `int64_t`?